orttraining/orttraining/test/python/orttraining_run_multiple_choice.py

```diff
@@ -91,22 +91,36 @@ def setUp(self):
         self.logging_steps = 10
 
     def test_bert_with_swag(self):
-        expected_acc = 0.7752174347695692
-        expected_loss = 0.5848011111033019
-
-        results = self.run_multiple_choice(model_name="bert-base-cased", task_name="swag", fp16=False)
-        assert_allclose(results['acc'], expected_acc)
-        assert_allclose(results['loss'], expected_loss)
+        expected_acc = 0.7640207937618715
+        expected_loss = 0.6234657892213054
+        results_per_api = dict()
+        for use_new_api in [False, True]:
+            results = self.run_multiple_choice(model_name="bert-base-cased", task_name="swag", fp16=False, use_new_api=use_new_api)
+            # assert_allclose(results['acc'], expected_acc)
+            # assert_allclose(results['loss'], expected_loss)
+            results_per_api[use_new_api] = results
+
+        for key in ['acc', 'loss']:
+            assert_allclose(results_per_api[False][key], results_per_api[True][key])
 
     def test_bert_fp16_with_swag(self):
-        expected_acc = 0.7765170448865341
-        expected_loss = 0.5851960482903881
+        # larger batch can be handled with mixed precision
+        self.train_batch_size = 32
+
+        expected_acc = 0.7482255323402979
+        expected_loss = 0.6665529619455844
 
-        results = self.run_multiple_choice(model_name="bert-base-cased", task_name="swag", fp16=True)
-        assert_allclose(results['acc'], expected_acc)
-        assert_allclose(results['loss'], expected_loss)
+        results_per_api = dict()
+        for use_new_api in [False, True]:
+            results = self.run_multiple_choice(model_name="bert-base-cased", task_name="swag", fp16=True, use_new_api=use_new_api)
+            assert_allclose(results['acc'], expected_acc)
+            assert_allclose(results['loss'], expected_loss)
```

Comment:
```suggestion
```
Won't this be included into `results_per_api` which will then be tested by `verify_old_and_new_api_are_equal`

Reply:
this test is to ensure run results match the golden number.